### PR TITLE
css-popup-menu-task

### PR DESCRIPTION
**Pull request by PAtrofimov**
### css-popup-menu-task

[Demo](https://patrofimov.github.io/css-popup-menu/) |
[Code base](https://github.com/PAtrofimov/css-popup-menu)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/patrofimov/css-popup-menu/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                  <li class="sub-menu-more-item"><a href="#">Sub10</a></li>
+                  <a class="other-products" href="#">Even more from Google</a>
+               </ul>
+            </ul>
+         </li>
+         <li class="main-menu-item"><a href="#">Home</a></li>
+         <li class="main-menu-item"><a href="#">Login</a></li>
+      </ul>
+   </body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/css-popup-menu/index.html]: Add an empty line at the end of every file.

File: submissions/patrofimov/css-popup-menu/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+         <li class="main-menu-item folder">
+            <a class="main-item" href="#" tabindex="1" >Menu</a> 
+            <ul class="sub-menu">
+               <li class="sub-menu-item"><a href="#">Icon1</a></li>
+               <li class="sub-menu-item"><a href="#">Icon2</a></li>
+               <li class="sub-menu-item"><a href="#">Icon3</a></li>
+               <li class="sub-menu-item"><a href="#">Icon4</a></li>
+               <li class="sub-menu-item"><a href="#">Icon5</a></li>
+               <li class="sub-menu-item"><a href="#">Icon6</a></li>
+			   <li class="sub-menu-item"><a href="#">Icon7</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/css-popup-menu/index.html]: Use prettier for formatting your code.

File: submissions/patrofimov/css-popup-menu/style.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.sub-menu-item,
+.sub-menu-more-item,
+.main-menu-item {
+	margin-right: 10px;
+	margin-left: 10px;
+}
+
+a {
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/css-popup-menu/style.css]: Avoid styling by tag. Better use class.

**Comment by PAtrofimov** [comment]
Fixed everything.

**Comment by A-Ostrovnyy** [comment]
Your demo must contain images. Images must have outline on `:focus` state. At this moment menu link must be image and have outline on `:focus` state too.
When the menu icon on focus I press in Space or Enter button menu must open. All interactive elements should have `:hover` and `:focus` effect.
Please fix these issues.

**Comment by PAtrofimov** [comment]
Fixed focus and hover states. Pressing in Space button menu open.

**Comment by A-Ostrovnyy** [comment]
Nice work!

**Pull request merged**